app/src/componentLibrary/attributeList/attributeList.jsx

```diff
@@ -111,18 +130,24 @@ export const AttributeList = ({
     onChange(newAttributes);
   };
 
-  const isNewAttribute = (attribute) => !attribute.value;
-
   const createCancelEditHandler = (index) => () => {
     const newAttributes = [...attributes];
     if (isNewAttribute(attributes[index])) {
       newAttributes.splice(index, 1);
+
+      handleAddNewAttrFocus();
     } else {
       const { edited, ...attribute } = newAttributes[index];
       newAttributes[index] = attribute;
+
+      scheduleUpdate(() =>
+        handleChangeFocus(
+          getAttributesMapValue(getAttributeUniqueKey(attributes[index]), attributeWrapperRef),
```

Comment:
Is it possible to use `attribute` here? Looks like it is the same with `attributes[index]`.